msbuild/tests/Xamarin.iOS.Tasks.Tests/ProjectsTests/TVOS/TVApp.cs

```diff
@@ -2,18 +2,18 @@
 
 namespace Xamarin.iOS.Tasks
 {
-	[TestFixture("TV", "iPhone")]
-	[TestFixture("TVSimulator", "iPhoneSimulator")]
```

Comment:
SO, the file is called TVOS/TVApp, and we are deleting the TV and TVSimulator, is that correct?

Reply:
This is best explained by viewing the relevant commit only: https://github.com/xamarin/xamarin-macios/pull/7877/commits/b1fcce426f6055b76f87426a606763a139dc6042, in short it's because I changed the OutputPath in the csproj to be iPhone[Simulator] instead of TV[Simulator].